webcam.go

```diff
@@ -106,6 +106,26 @@ func (w *Webcam) GetSupportedFrameSizes(f PixelFormat) []FrameSize {
 	return result
 }
 
+// Returns supported frame rates for a given image format and frame size
+func (w *Webcam) GetSupportedFramerates(fp PixelFormat, width uint32, height uint32) []FrameRate {
+	result := make([]FrameRate, 0)
+
+	var index uint32
+	var err error
+
+	for index = 0; err == nil; index++ {
```

Comment:
can you explain this for loop? Is this saying to keep looping until we find an error? What if all we get are errors? Are the only errors that are returned by this ones we don't care to report?

Reply:
got it, I think it would be slightly easier to read if you did a `for err == nil` to make it a while loop and increment an index variable outside of this. using the index in here makes it seem like we are iterating through some range